### ISSUE TITLE
Oradb module confused when installdb and installasm instantiated on the same node

### ISSUE THREAD

**Issue by carroarmato0** [biemond/biemond-oradb#54]
It's probably not meant to work this way, but I happen to use  installdb and installasm on the same node (mostly for testing purposes).

This causes both installations to happen and work correctly during the first deploy, but afterwards during the next runs, the module gets confused because it makes a comparison between the different installations and decides to re-do the installation of one of them over and over again:

```
==> oracle12asm: Info: Loading facts
==> oracle12asm: Info: oracle_exists: oracle_exists /oragrid/product/12.1/grid;
==> oracle12asm: Info: oracle_exists: oracle_exists compare /oragrid/product/12.1/grid; with /oragrid/product/12.1/grid
==> oracle12asm: Info: oracle_exists: oracle_exists return true
==> oracle12asm: Info: oracle_exists: oracle_exists /oragrid/product/12.1/grid;
==> oracle12asm: Info: oracle_exists: oracle_exists compare /oragrid/product/12.1/grid; with /oradb/product/12.1/db
==> oracle12asm: Info: oracle_exists: oracle_exists return false
==> oracle12asm: Notice: Compiled catalog for oracle12asm.vagrantup.com in environment production in 2.18 seconds
==> oracle12asm: Info: Applying configuration version '1433334088'
==> oracle12asm: Notice: oradb::installdb /oradb/product/12.1/db does not exists
==> oracle12asm: Notice: /Stage[main]/Vdab_oracledb/Oradb::Installdb[12.1.0.2_Linux-x86-64]/Notify[oradb::installdb /oradb/product/12.1/db does not exists]/message: defined 'message' as 'oradb::installdb /oradb/product/12.1/db does not exists'
```

So basically this is the problem:
```
==> oracle12asm: Info: oracle_exists: oracle_exists compare /oragrid/product/12.1/grid; with /oradb/product/12.1/db
```

**Comment by biemond** [comment]
Hi,

it should work perfectly can  you take look at this 
CentOS 6.6 vagrant box with Oracle Database 11.2.0.4 on NFS ASM ASM vagrant box
https://github.com/biemond/biemond-oradb-vagrant-11.2-ASM

thanks

**Comment by carroarmato0** [comment]
The link you provided only uses  installasm.

My issue happens when you use both  installdb and installasm at the same time.

**Comment by biemond** [comment]
on 
https://github.com/biemond/biemond-oradb-vagrant-11.2-ASM/blob/master/puppet/manifests/site.pp#L211

I install asm and db on
https://github.com/biemond/biemond-oradb-vagrant-11.2-ASM/blob/master/puppet/manifests/site.pp#L264

but the directories are very crucial

oraInventory_dir:         '/app'

# global oracle vars
oracle_base_dir:          '/app/oracle'
oracle_home_dir:          '/app/oracle/product/11.2/db'

# global OS vars
oracle_os_user:           "oracle"
oracle_os_group:          "dba"
oracle_download_dir:      "/var/tmp/install"
oracle_source:            "/software"

grid_base_dir:            '/app/grid'
grid_home_dir:            '/app/grid/product/11.2/grid'

grid_os_user:             "grid"
grid_os_group:            "asmdba"

**Comment by carroarmato0** [comment]
using a common oraInventory proved to have solved the issue for me. Thank you.
